libpod/rootless_cni_linux.go

```diff
@@ -25,7 +25,7 @@ import (
 
 // Built from ../contrib/rootless-cni-infra.
 var rootlessCNIInfraImage = map[string]string{
-	"amd64": "quay.io/libpod/rootless-cni-infra@sha256:304742d5d221211df4ec672807a5842ff11e3729c50bc424ea0cea858f69d7b7", // 3-amd64
+	"amd64": "quay.io/luap99/rootless-cni-infra@sha256:fcc48fa68be51ae177d8c4c865c448d93531fa874362b710f2d64b117687268c", // 5-amd64
```

Comment:
@mheon I need the image before we merge. I have to update this line with the correct digest.

Reply:
Ack, let me do that.

Reply:
Built and pushed as `sha256:adf352454666f7ce9ca3e1098448b5ee18f89c4516471ec99447ec9ece917f36`

Reply:
Thanks